micromagneticmodel/abstract/container.py

```diff
@@ -486,3 +486,7 @@ def _repr_latex_(self):
                         reprlatex += termlatex
 
         return f"${reprlatex}$"
+
+    def get(self, *, type):
+        """Return a list of all terms of type ``type`` in the container."""
+        return [term for term in self if isinstance(term, type)]
```

Comment:
What we mostly need (at least inside `oommfc`) is access to the first element. `Container` currently does not implement `__getitem__` so that is not directly possible. I can add that if you think it would be an advantage

Reply:
I think it should behave as a Sequence as much as possible. `__getitem__` should be sufficient since `__iter__` for unpacking is derived automatically.
```python
def __getitem__(self, index):
    return self._terms[index]
```